sdk/eventhubs/microsoft-azure-eventhubs/pom.xml

```diff
@@ -17,7 +17,7 @@
   <version>3.4.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs;current} -->
 
   <name>Microsoft Azure SDK for Event Hubs</name>
-  <description>Please note, a newer package azure-messaging-eventhubs for Azure Event Hubs is available at https://search.maven.org/artifact/com.azure/azure-messaging-eventhubs as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/java/migrate/eh for more details.
+  <description>Please note, a newer package azure-messaging-eventhubs for Azure Event Hubs is available at https://central.sonatype.com/search?namespace=com.azure&amp;name=azure-messaging-eventhubs as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/java/migrate/eh for more details.
```

Comment:
Update `&amp;` with `&` in all pom.xml files.

Reply:
I believe the description tag is for HTML and I believe `&` is reserved